FluentTc.Tests/AcceptanceTests.cs

```diff
@@ -442,17 +442,42 @@ public void RunBuildConfiguration_OnAgentNameWithParameters()
             var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller, buildConfigurationRetriever, agentsRetriever);
 
             // Act
-            connectedTc.RunBuildConfiguration(having, onAgent, p => p.Parameter("param1", "value1"));
+            var build = connectedTc.RunBuildConfiguration(having, onAgent, p => p.Parameter("param1", "value1"));
 
             // Assert
             A.CallTo(
                 () =>
-                    teamCityCaller.Post("<build>\r\n<buildType id=\"bt2\"/>\r\n<agent id=\"9\"/>\r\n<properties>\r\n<property name=\"param1\" value=\"value1\"/>\r\n</properties>\r\n</build>\r\n",
-                    HttpContentTypes.ApplicationXml, "/app/rest/buildQueue", A<string>.Ignored))
+                    teamCityCaller.PostFormat<BuildModel>("<build>\r\n<buildType id=\"bt2\"/>\r\n<agent id=\"9\"/>\r\n<properties>\r\n<property name=\"param1\" value=\"value1\"/>\r\n</properties>\r\n</build>\r\n",
+                    HttpContentTypes.ApplicationXml, HttpContentTypes.ApplicationJson, "/app/rest/buildQueue", A<object[]>.Ignored))
                         .MustHaveHappened(Repeated.Exactly.Once);
         }
 
         [Test]
+        public void RunBuildConfiguration_BuildResponse()
+        {
+            // Arrange
+            Action<IBuildConfigurationHavingBuilder> having = _ => _.Name("FluentTc");
+            var teamCityCaller = CreateTeamCityCaller();
+            var buildConfigurationRetriever = A.Fake<IBuildConfigurationRetriever>();
+
+            A.CallTo(() => buildConfigurationRetriever.GetSingleBuildConfiguration(having))
+                .Returns(new BuildConfiguration { Id = "bt2" });
+            A.CallTo(
+                () =>
+                    teamCityCaller.PostFormat<BuildModel>(A<string>.Ignored,
```

Comment:
@nex-54 can you replace "A<string>.Ignored" with the actual string that you expect it to be?